libraries/GCS_MAVLink/GCS.h

```diff
@@ -771,6 +771,7 @@ class GCS_MAVLINK
                                                      const float roll,
                                                      const float pitch,
                                                      const float yaw,
+                                                     const float covariance[],
```

Comment:
Should we perhaps change this to a float covariance[21]?  I think it doesn't really matter but it would stop someone from being able to pass a very short array and then having the VisualOdom code read outside the bounds of the array

Reply:
Yes. I will fix this. Thank you